xz.spec

```diff
@@ -1,13 +1,14 @@
-### RPM external xz 5.0.3__5.1.2alpha
-%define generic_version 5.0.3
+### RPM external xz 5.2.1__5.1.2alpha
+%define generic_version 5.2.1
 %define fcarm_version 5.1.2alpha
-%define tag 931d2d5
+%define tag dec11497a71518423b5ff0e759100cf8aadf6c7b
 %define branch cms/v%generic_version
-%define github_user cms-externals
+%define github_user degano
```

Comment:
This should be __cms-externals__. Also there is no need for `5.1.2alpha`. Just build `5.2.1` on all archs and remove ARMv7 bits (not needed).